### PR TITLE
Update Microsoft.Windows.Compatibility README.md

### PR DESCRIPTION
**Pull request by KlausLoeffelmann**

### CODE REVIEW

File: src/Microsoft.Windows.Compatibility/src/README.md [hunk 1]
```diff
@@ -1,9 +1,106 @@
...
+
+### .NET CLI
+```
+dotnet add package Microsoft.Windows.Compatibility --version [your_version_here]
+```
+
+### Prerequisites
+
+- .NET Core 2.0-3.1
+- .NET 5-.NET 8+
```
Comment (Tanya-Solyanik) [inline src/Microsoft.Windows.Compatibility/src/README.md:22]: Docs site lists .NET versions with a comma - 
<html>
<body>


.NET | Core 1.0, Core 1.1, Core 2.0, Core 2.1, Core 2.2, Core 3.0, Core 3.1, 5, 6, 7, 8
-- | --



</body>
</html>

File: src/Microsoft.Windows.Compatibility/src/README.md [hunk 2]
```diff
@@ -1,9 +1,106 @@
...
+```csharp
+using Microsoft.Win32;
+
+class Program
+{
+    static void Main()
+    {
+        RegistryKey key = Registry.CurrentUser.CreateSubKey("Software\MyApp");
+        key.SetValue("MySetting", "MyValue");
+        key.Close();
```
Comment (Tanya-Solyanik) [inline src/Microsoft.Windows.Compatibility/src/README.md:41]: We usually do `using`, if you close explicitly, you need a try/catch. This applies to all samples.

File: src/Microsoft.Windows.Compatibility/src/README.md [hunk 3]
```diff
@@ -1,9 +1,106 @@
...
+
+#### C#
+```csharp
+using System.IO.Ports;
+
+class Program
+{
+    static void Main()
+    {
+        SerialPort sp = new SerialPort("COM1", 9600);
```
Comment (Tanya-Solyanik) [inline src/Microsoft.Windows.Compatibility/src/README.md:69]: I would prefer `port` to `sp`.

File: src/Microsoft.Windows.Compatibility/src/README.md [hunk 4]
```diff
@@ -1,9 +1,106 @@
...
+For more in-depth tutorials and API references, you can check the following resources:
+
+- [NuGet Gallery | Microsoft.Windows.Compatibility](https://nuget.org/packages/Microsoft.Windows.Compatibility/)
+- [Use the Windows Compatibility Pack to port code - .NET](https://learn.microsoft.com/en-us/dotnet/core/porting/windows-compat-pack)
+- [Announcing the Windows Compatibility Pack for .NET Core](https://devblogs.microsoft.com/dotnet/announcing-the-windows-compatibility-pack-for-net-core/)
+- [Installing NuGet client tools | Microsoft Learn](https://learn.microsoft.com/en-us/nuget/consume-packages/install-use-packages-nuget-cli)
+
+## Feedback
+
+We value your feedback! Here are ways to get in touch with us:
```
Comment (Tanya-Solyanik) [inline src/Microsoft.Windows.Compatibility/src/README.md:102]: This is a nice phrase, we can use it in the System.Drawing.Common readme too!

File: src/Microsoft.Windows.Compatibility/src/README.md [hunk 5]
```diff
@@ -1,9 +1,106 @@
...
+    End Sub
+End Module
+```
+
+## Additional Documentation
+
+For more in-depth tutorials and API references, you can check the following resources:
+
+- [NuGet Gallery | Microsoft.Windows.Compatibility](https://nuget.org/packages/Microsoft.Windows.Compatibility/)
+- [Use the Windows Compatibility Pack to port code - .NET](https://learn.microsoft.com/en-us/dotnet/core/porting/windows-compat-pack)
```
Comment (Tanya-Solyanik) [inline src/Microsoft.Windows.Compatibility/src/README.md:96]: Please remove `en-us` from the links.

File: src/Microsoft.Windows.Compatibility/src/README.md [hunk 6]
```diff
@@ -1,9 +1,106 @@
...
 
-### How to add a new dependency
-1. Add the package version to the repository's Versions.props file.
-2. Add the package dependency to the repository's Version.Details.xml file. If the dependency doesn't come from a repository with a direct subscription to (i.e. winforms and runtime), add the `CoherentParentDependency` attribute to the dependency.
-3. Add the package reference into the Microsoft.Windows.Compatibility.csproj file and use the version just defined in the Versions.props file.
-4. If a `CoherentParentDependency` attribute was required, a dependency must be added to the repository dependency chain as well. I.e. to add a new dependency from runtime, a dependency must also be added in winforms and wpf's Version.Details.xml file.
\ No newline at end of file
+## Getting Started
+
+To start using the Microsoft.Windows.Compatibility package, you'll first need to install it via NuGet Package Manager, Package Manager Console, or by editing your project file.
```
Comment (Tanya-Solyanik) [inline src/Microsoft.Windows.Compatibility/src/README.md:7]:
```suggestion
To start using the `Microsoft.Windows.Compatibility` package, you'll first need to install it via NuGet Package Manager, Package Manager Console, or by editing your project file.
```

File: src/Microsoft.Windows.Compatibility/src/README.md [hunk 7]
```diff
@@ -1,9 +1,106 @@
...
+
+### Prerequisites
+
+- .NET Core 2.0-3.1
+- .NET 5-.NET 8+
+- .NET Standard 2.0 or higher
+
+## Usage
+
+After installing the package, you can access Windows-specific APIs just like you would in a .NET Framework application. Below are some examples in both C# and VB:
```
Comment (Tanya-Solyanik) [inline src/Microsoft.Windows.Compatibility/src/README.md:27]: It might be helpful to list all public namespaces that are shipped in this package.

**Comment by Tanya-Solyanik** [comment]
You will cherry-pick this change to release/8.0 branch, right?

File: src/Microsoft.Windows.Compatibility/src/README.md (hunk 7 above)
Comment (KlausLoeffelmann) [reply src/Microsoft.Windows.Compatibility/src/README.md:27]: Yeah, I don't know. That's rather the job of the docs.

**Review by Tanya-Solyanik — APPROVED** [review]

**Review by Tanya-Solyanik — APPROVED** [review]

File: src/Microsoft.Windows.Compatibility/src/README.md [hunk 8]
```diff
@@ -1,9 +1,108 @@
...
+### .NET CLI
+```
+dotnet add package Microsoft.Windows.Compatibility --version [your_version_here]
+```
+
+### Prerequisites
+
+- .NET Core 2.0+
+- .NET 5+
+- .NET Standard 2.0 or higher
```
Comment (ViktorHofer) [inline src/Microsoft.Windows.Compatibility/src/README.md:23]: Please remove these three sections. These are already part of the UI of nuget.org and tooling:

image

**Comment by ViktorHofer** [comment]
@KlausLoeffelmann you did accidentally update the repo documentation for the Microsoft.Windows.Compatibility package which targets devs that need to update the library in this repository.

Instead, this should have gone into the src directory and named PACKAGE.md. Please see https://github.com/dotnet/runtime/pull/91210 for an example.

File: src/Microsoft.Windows.Compatibility/src/README.md (hunk 8 above)
Comment (KlausLoeffelmann) [reply src/Microsoft.Windows.Compatibility/src/README.md:23]: That would then also count for `System.Drawing.Common`, I suppose, which did not have a readme, either?

Comment (ViktorHofer) [reply src/Microsoft.Windows.Compatibility/src/README.md:23]: Did you already add a package readme for System.Drawing.Common? Yes, probably applies to that library as well.

**Comment by KlausLoeffelmann** [comment]
Ah, shoot. OK, I am fixing that right now. 
Will apply probably for the `System.Drawing.Common` package, either.

**Pull request merged**